ipfx/nwb_reader.py

```diff
@@ -207,16 +343,19 @@ class NwbMiesReader(NwbReader):
     """
     Reads data from the MIES generated NWB file
     """
+
     def __init__(self, nwb_file):
-        NwbReader.__init__(self, nwb_file)
+        NwbReader.__init__(self, nwb_file, nwb_major_version=1)
 
     def get_sweep_data(self, sweep_number):
 
         with h5py.File(self.nwb_file, 'r') as f:
-            sweep_response = f['acquisition']['timeseries']["data_%05d_AD0" % sweep_number]
+            sweep_response = f[self.acquisition_path]["data_%05d_AD0" %
+                                                      sweep_number]
             response = sweep_response["data"].value
             hz = 1.0 * sweep_response["starting_time"].attrs['rate']
-            sweep_stimulus = f['stimulus']['presentation']["data_%05d_DA0" % sweep_number]
+            sweep_stimulus = f[self.stimulus_path]["data_%05d_DA0" %
+                                                   sweep_number]
             stimulus = sweep_stimulus["data"].value
 
             if 'unit' in sweep_stimulus["data"].attrs:
```

Comment:
Could use get_long_unit_name here as well

Reply:
@sgratiy I've seen that you have already done that in your PR. If you want me to include the changes in my PR as well just ping me.